pkg/kubelet/kubelet.go

```diff
@@ -679,7 +679,8 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 			klet.runtimeCache,
 			kubeDeps.RemoteRuntimeService,
 			kubeDeps.RemoteImageService,
-			hostStatsProvider)
+			hostStatsProvider,
+			utilfeature.DefaultFeatureGate.Enabled(features.DisableAcceleratorUsageMetrics))
```

Comment:
just curious, why are we only doing this for the CRI stats provider (and not also the regular [dockershim] stats provider)?

Reply:
regular stats provider is disabled by MetricSet. It doesn't do any transformations